scenes/controller/unlock_controller/unlock_controller.gd

```diff
@@ -95,6 +98,8 @@ func _handle_npc_event_interacted(npc_id: String, npc_event_id: String, option:
 
 
 func _handle_land_event(observed_id: String) -> void:
+	if sfx_unlock:
+		Audio.play_sfx(sfx_unlock, 0.0)
```

Comment:
is it intended for this to play every 5 seconds ? (worker role explorer generates land) 👀 

or should it be better to trigger only on actual unlocks?

Reply:
Oh yeah it should be on unlocks, I'll move it to the right place